src/ISO3166.php

```diff
@@ -137,12 +152,12 @@ public function getIterator()
     private function getBy($key, $value)
     {
         foreach ($this as $country) {
-            if (0 === strcasecmp($value, $country[$key])) {
+            if ($value === $country[$key]) {
                 return $country;
             }
         }
 
-        throw new \OutOfBoundsException('ISO 3166-1 does not contain: ' . $value);
+        throw new \OutOfBoundsException(sprintf('ISO 3166-1 does not contain: %s', $value));
```

Comment:
Me nitpicking. The message needs to be re-formatted to say that the value is not contain in the collection. Let's say I create a collection of Asian countries. If I do $collection->getByAlpha2('fr'); <- it will trigger the OutOfBoundsException **BUT** FR is a valid ISO3166 code